FableAnniversary.asl

```diff
@@ -43,9 +44,22 @@ startup {
 }
 
 isLoading {
-	return current.isLoading > 0;
+	if(current.isLoading > 0){
+		return current.isLoading > 0;
+	}
+
+	if(current.isLoadingSave > 0){
+		return current.isLoadingSave > 0;
+	}
+
+	if(current.isLoading == 0 && current.isLoadingSave == 0){
+		return current.isLoading > 0;
+		return current.isLoadingSave > 0;

```

Comment:
Unreachable statement

Reply:
I spoke with Litigacy on 4/20/21 to clarify this comment.

In order to reach this line, the condition on line 55 must be true.  However, in that case, it will hit the return on 56 first and stop executing before ever reaching 57. The code here will never be run.